plex_set_tracks.py

```diff
@@ -221,13 +219,7 @@ def getSeasonsFromUser(show):
 
         # Display season numbers 
         print("You have the following seasons of '%s': [" % (show.title), end="")
-        isFirstSeason = True
-        for season in seasonNums:
-            if isFirstSeason == True:
-                print("%d" % (season), end="")
-                isFirstSeason = False
-            else:
-                print("|%d" % (season), end="")
+        print("|".join([str(s) for s in seasonNums]))
```

Comment:
You'll need to stop it from printing a newline char here: `print("|".join([str(s) for s in seasonNums]), end="")`